### DISCUSSION TITLE
how to compress my data into a .pt file?

### DISCUSSION THREAD

**Discussion by Wenjian-Ma** [pyg-team/pytorch_geometric#5653] [Q&A]
Here, data_list denotes a set of molecular graph, which can be compress into .pt file by "torch.save((data, slices), self.processed_paths[0])". However, I want to compress protein graph and molecular graph (i.e., data_prot_list) into one .pt file, anyone can tell me the solution of this question? Thanks so much!
![image](https://user-images.githubusercontent.com/35370756/195119465-73334012-2c33-4d9c-b326-c48884bc49f7.png)

**Comment by wsad1** [comment]
Maybe you can elaborate some more. But `self.collate` combines the list of `data` objects into one large data object, and  then a `.pt` file gets saved for data and slices.
